debugTestImage.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env tsx
+/**
+ * @file like runTestImages.ts, but for a single proposal,
+ * and leaves the chain running
+ */
+import { execSync } from 'node:child_process';
+import { parseArgs } from 'node:util';
+import { imageNameForProposal, readOneProposal } from './common';
+
+const options = {
+  match: { short: 'm', type: 'string' },
+} as const;
+const { values } = parseArgs({ options });
+
+const proposal = readOneProposal(values.match!);
```

Comment:
I'm curious: what does that (`values.match!`) compile to?

Is this another case where we're outside a .ts dialect where the difference between .ts and .js is "erase the types"?

Reply:
it's elided in the JS. it just tells TS that the we're sure the value isn't undefined.